lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -380,24 +380,18 @@
                 createTeamsListView: function(options) {
                     var topic = options.topic,
                         collection = options.collection,
-                        myTopicTeamsCollection = new TeamCollection(
-                            this.context.userInfo.teams,
-                            {
-                                teamEvents: this.teamEvents,
-                                course_id: this.context.courseID,
-                                username: this.context.userInfo.username,
-                                topic_id: topic.get('id'),
-                                perPage: 5,
-                                parse: true,
-                                url: this.context.myTeamsUrl
+                        topicTeams = _.filter(
+                            this.context.userInfo.teams.results,
```

Comment:
I know `this.context.userInfo.teams` is a serialized list of the user's teams, but what's `.results`?